### PR TITLE
Homepage styling

### PR DESCRIPTION
**Pull request by sagarbadu**
HomePage displays the courses. Has simple styling.

### CODE REVIEW

**Comment by n-grounds** [comment]
Seems strange to be removing the src/components/Assignments.js page/file, was that intended?

**Comment by sagarbadu** [comment]
> Seems strange to be removing the src/components/Assignments.js page/file, was that intended?

Yes sir, for some reason it was giving an error. I think maybe there was something wrong with materialui installation. I just wanted to finish modifying the homepage a little bit. That's why I removed the Assignment.js file for a moment, but I am trying to fix the error right now by  re-installing material ui.

**Comment by logan-giese** [comment]
The Assignments.js problems have been patched in dev so I restored it here.

File: src/components/Home.js [hunk 1]
```diff
@@ -1,8 +1,55 @@
...
+import { Container, Card, Row, Col } from "react-bootstrap";
+import "bootstrap/dist/css/bootstrap.min.css";
+
+const Home = () => {
+  const [courses, setCourses] = useState([]);
+
+  const getCourses = async () => {
+    setCourses(await CourseService.getCourses());
+  };
+  getCourses();
```
Comment (logan-giese) [inline src/components/Home.js:13]: In the future it's generally better to load courses with useEffect() to reduce the number of calls to the API.

File: src/components/Home.js [hunk 2]
```diff
@@ -1,8 +1,55 @@
...
+
+  return React.createElement(
+    "div",
+    { id: "hello", className: "dummyClass" },
```
Comment (logan-giese) [inline src/components/Home.js:17]: Do id=hello and className=dummyClass actually do anything?  If not, it would be better to remove them.

**Review by logan-giese — APPROVED** [review]
Tested and working great!

**Pull request merged**